PAM_app/app/src/main/java/com/example/pam_app/WelcomeActivity.kt

```diff
@@ -0,0 +1,37 @@
+package com.example.pam_app
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Button
+import android.widget.TextView
+import java.text.SimpleDateFormat
+import java.util.*
+
+class WelcomeActivity : AppCompatActivity() {
+
+    lateinit var main_page_button: Button
+    lateinit var login_button: Button
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_welcome)
```

Comment:
Change init of activity view to android view binding https://developer.android.com/topic/libraries/view-binding